pkg/machine-info/machine_info.go

```diff
@@ -45,7 +46,8 @@ func GetMachineInfo(nvmlInstance nvidianvml.Instance) (*apiv1.MachineInfo, error
 		Hostname:                hostname,
 		Uptime:                  metav1.NewTime(time.Unix(int64(pkghost.BootTimeUnixSeconds()), 0)),
 
-		CPUInfo: GetMachineCPUInfo(),
+		CPUInfo:     GetMachineCPUInfo(),
+		NetworkInfo: GetMachineNetwork(),
```

Comment:
GetMachineNetwork only returns the public ip, we also need the private ip.
we should add a new structure to collect all physical nic devices including the IB devices.

Reply:
Shall we report the private ip specified when login for now?